ci_framework/roles/repo_setup/tasks/install.yml

```diff
@@ -7,6 +7,12 @@
     - tmp
     - artifacts/repositories
 
+- name: Make sure git package is installed
```

Comment:
please make sure to add
```YAML
tags:
  - packages
  - bootstrap
```
so that it will be picked up during those runs. One may argue "this should be part of the ci_setup role" - I'm not sure whether any of the other roles are consuming git at some point?

Reply:
Add tags. For timebeing, I am keeping it here. as we are doing git operation in repo_setup role.

Reply:
:+1: